tests/integration/test_engine_integration.py

```diff
@@ -19,7 +19,7 @@
 
```

Comment:
Note: I did not add any tests that check if the object returned from the backend has a particular shape, but we could add one for `gaussianbackend.measure_fock` I suppose?

Reply:
Added in the measure_* backend tests.